submissions/jeffersonMormah/php/hobbyArray.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+$firstNames = ["Jeff","Michelle","Matthew","Kelly","Wayne","Favor","Juliet","Daniel"];
+$resHobbies = ["Coding","Music","Football","Dancing","Trading","Singing","Fashion","Acting"];
+
+foreach(data_array($firsNames,$resHobbies) as $firstname => $hobby){
```

Comment:
what does `data_array()` do?

Reply:
I used data_array as a function to create an array that will use the elements from the above parameters that were passed in the function. So a better function for this purpose was the array_combine function.